_analyzers/supported-analyzers/phone-analyzers.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Phone number
+parent: Analyzers
+nav_order: 140
+---
+
+# Phone number analyzers
+
+The `analysis-phonenumber` plugin provides analyzers and tokenizers for parsing phone numbers. A dedicated analyzer is required because parsing phone numbers is a non-trivial task (even though it might seem trivial at first glance). For common misconceptions regarding phone number parsing, see [Falsehoods programmers believe about phone numbers](https://github.com/google/libphonenumber/blob/master/FALSEHOODS.md).
+
+
+OpenSearch supports the following phone number analyzers:
+
+* [`phone`](#the-phone-analyzer): An [index analyzer]({{site.url}}{{site.baseurl}}/analyzers/index-analyzers/) to use at indexing time.
+* [`phone-search`](#the-phone-search-analyzer): A [search analyzer]({{site.url}}{{site.baseurl}}/analyzers/search-analyzers/) to use at search time.
+
+Internally, the plugin uses the [`libphonenumber`](https://github.com/google/libphonenumber) library and follows its parsing rules.
+
+The phone number analyzers are not meant to find phone numbers in larger texts. Instead, you should use them on fields that only contain phone numbers.
+{: .note}
+
+## Installing the plugin
+
+Before you can use the phone number analyzers, you must install the `analysis-phonenumber` plugin by running the following command:
+
+```sh
+./bin/opensearch-plugin install analysis-phonenumber
+```
+
+## Specifying a default region
+
+You can optionally specify a default region for parsing phone numbers by providing the `phone-region` parameter within the analyzer. Valid phone regions are represented by ISO 3166 country codes. For more information, see [List of ISO 3166 country codes](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes).
+
+When tokenizing phone numbers containing the international calling prefix `+`, the default region is irrelevant. However, for phone numbers that use a national prefix for international numbers (for example, `001` instead of `+1` to dial Northern America from most European countries), the region needs to be provided. You can also properly index local phone numbers with no international prefix by specifying the region.
+
+## Example
+
+The following request creates an index containing one field that ingests phone numbers for Switzerland (region code `CH`):
+
+```json
+PUT /example-phone
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "phone-ch": {
+          "type": "phone",
+          "phone-region": "CH"
+        },
+        "phone-search-ch": {
+          "type": "phone-search",
+          "phone-region": "CH"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "phoneNumber": {
```

Comment:
btw: do examples usually use `camelCase`, `snake_case` or `kebab-case`? this one currently uses `kebab-case` for the analyzer names (the actual `phone-search` analyzer does so too) and index name but `camelCase` for the field here (sorry, don't know how i ended up with this mix in this example)

Reply:
In general, OpenSearch uses `snake_case` in requests.

Reply:
Ideally, it should all be snake case, so yes, `phone_number` is preferred. That said, even our own alerting plugin uses camel case. I think it would be nice to change to snake case 😄